Update form-field-label-descriptive-cc0f0a.md (#2345)
auto-wcag/auto-wcag: _rules/form-field-label-descriptive-cc0f0a.md

Reviewer: ```suggestion
The `label` element has a visible relationship with the `input` element and describes it.
```

Reviewer: ```suggestion
The `label` elements visually related to their respective `input` elements. The `label` elements, are not descriptive enough (because they are repeated over several fields). However, the headings also works as a visual label that differentiates the purpose of the otherwise identically named form fields.
```

Author: ```suggestion
The `label` elements are visually related to their respective `input` elements. The `label` elements are not descriptive enough (because they are repeated over several fields). However, the headings also provide visual labels that differentiate the purpose of the otherwise identically named form fields.
```

Reviewer: This now passes. I think we can delete this example.

Reviewer 2: > and where both the element and the [programmatic label][] are [visible][].

the programmatic label is still not visible, so it's still inapplicable.

and where both the element and the [programmatic label][] are [visible][].

## Expectation 1

Each test target has a label, and the label is descriptive.

## Expectation 2

> Each test target, together with its [visual context][], describes the purpose of the associated element.

Reviewer 3: ```suggestion
Each test target's label, together with its [visual context][], describes the purpose of the associated element.
```

Author: As discussed in today's meeting, I'll be removing this and refactoring expectation 1 into two separate expectations instead. We will still need to come up with a better definition for label, however.

and where both the element and the [programmatic label][] are [visible][].

## Expectation 1

> Each test target has at least one label.

Reviewer: ```suggestion
Each test target is positioned and styles so that it has a visual relationship with at least one [label][].

**Note**: Labels are not just adjacent text. They can include captions or headings for groups of controls, adjacent buttons, and may be non-textual content.
```

Author: ```suggestion
Each test target is positioned and styled so that it has a visual relationship with at least one [label][].

**Note**: Labels are not just adjacent text. They can include captions or headings for groups of controls or adjacent buttons, and they may be non-textual content.
```

## Expectation 2

> All of the test target's labels describe the purpose of the test target.

Reviewer: ```suggestion
Each test target's purpose is described by its labels (from expectation 1).
```
